html/src/vr.pug

```diff
@@ -15,42 +15,80 @@ html
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    | #[span.name(v-text="feed.displayName")] #[location(:location="feed.location[0]")]
-                        div(v-else-if="feed.type === 'Offline'" class="x-friend-item" :class="{ friend: feed.isFriend, favorite: feed.isFavorite }")
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    | #[span.name(v-text="feed.displayName")] ❌
-                        div(v-else-if="feed.type === 'Online'" class="x-friend-item" :class="{ friend: feed.isFriend, favorite: feed.isFavorite }")
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    | #[span.name(v-text="feed.displayName")] ✔️
-                        div(v-else-if="feed.type === 'Status'" class="x-friend-item" :class="{ friend: feed.isFriend, favorite: feed.isFavorite }")
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    | #[span.name(v-text="feed.displayName")] #[i.x-user-status(:class="userStatusClass(feed.status[0])")] {{feed.status[0].statusDescription}}
-                        div(v-else-if="feed.type === 'OnPlayerJoined'" class="x-friend-item" :class="{ friend: feed.isFriend, favorite: feed.isFavorite }")
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    | ▶️ #[span.name(v-text="feed.data")]
-                        div(v-else-if="feed.type === 'OnPlayerLeft'" class="x-friend-item" :class="{ friend: feed.isFriend, favorite: feed.isFavorite }")
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    | ◀️ #[span.name(v-text="feed.data")]
-                        div(v-else-if="feed.type === 'Location'" class="x-friend-item" :class="{ friend: feed.isFriend, favorite: feed.isFavorite }")
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    location(:location="feed.data")
+                    template(v-if="VRCXStorage.GetBool('VRCX_minimalFeed')")
```

Comment:
😨

Reply:
yea... I'm sure there is a better way, maybe RefreshVR() on setting change?